pysoa/common/schemas.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import unicode_literals
```

Comment:
The contents of this file were simply moved from pysoa/common/settings.py to resolve a circular import.